requirements.txt

```diff
@@ -1,5 +1,7 @@
 # Basic Python requirements for Garden Linux
 # that can be installed by Python pip.
 # Note: You should always prefer your local package manager.
-PyYAML
+json
 networkx
+yaml
```

Comment:
There is no pip package called yaml available. 
we should be able to remove this line, and just use the PyYAML